cloud-builder/generate_dependency_health_svg.py

```diff
@@ -0,0 +1,87 @@
+"""Generates a SVG by parsing json report.
+
+Task dependencyUpdates generates a json report. This script parses that file and
+generates a SVG showing percentage of up-to-date dependencies.
+
+Usage:
+> python cloud-builder/generate_dependency_health_svg.py <path_to_json_report> <output_svg_path>
+"""
+
+import json
+from sys import argv
+
+SVG_TEMPLATE = 'cloud-builder/dependency.svg'
+COLOR_RED = '#E05D44'
+COLOR_ORANGE = '#FFA100'
+COLOR_GREEN = '#43CC11'
+
+
+def get_status_color(health):
+    if health >= 67:
+        return COLOR_GREEN
+    elif 33 < health < 67:
+        return COLOR_ORANGE
+    else:
+        return COLOR_RED
```

Comment:
@gino-m @scolsen @dturner 
Thoughts on the current chosen range?

Reply:
I like the last one (5 ranges), since I think the impact of having older dependencies shouldn't be weighted so harshly - this way we'll have more yellow and green, only slipping into orange if we're really out of date.